src/metabase/sync/sync_metadata/sync_table_privileges.clj

```diff
@@ -14,7 +14,9 @@
    This is a cache of the data returned from `driver/table-privileges`, but it's stored in the database for performance."
   [database :- (mi/InstanceOf :model/Database)]
   (let [driver (driver.u/database->driver database)]
-    (when (driver/database-supports? driver :table-privileges database)
+    (when (and (not= :redshift driver)
+               ;; redshift does support table-privileges, but we don't want to sync it now
```

Comment:
sync last discussion you said that the primary reason we sync that is to enhance actions. and since redshift doesn't support actions, I don't think we have a good reason to have sync for it.

Reply:
> since redshift doesn't support actions, I don't think we have a good reason to have sync for it.

Yes exactly, I'm suggesting to leave that reason in the comment. In general when we make design decisions like this the rationale should be either easily recoverable from the code or left as a comment

Reply:
added a comment